src/main/java/com/coveo/nashorn_modules/Module.java

```diff
@@ -325,4 +329,28 @@ private static boolean isPrefixedModuleName(String module) {
   private static String[] getFilenamesToAttempt(String filename) {
     return new String[] {filename, filename + ".js", filename + ".json"};
   }
+
+  public NashornScriptEngine getEngine() {
```

Comment:
Hmm that's quite a bit of stuff that suddenly becomes public. Any specific reason for this?